package.json

```
@@ -18,7 +18,8 @@
     "clean": "rm -rf ./build ./plugin",
     "lint": "npm run lint-js & npm run lint-php",
     "lint-js": "wp-scripts lint-js blocks",
-    "lint-php": "composer run lint"
+    "lint-php": "composer run lint",
+    "test": "jest"
```

This can probably become the following and eliminate the need for dealing with `jest` ourselves.

```suggestion
    "test": "wp-scripts test-unit-js"
```

---

Thanks for the suggestion, I'll make the change :+1:

---

Nice to know about the `wp-scripts` tool btw :+1: